tests/ExporterTest.php

```diff
@@ -36,12 +36,8 @@ public function testFilter(): void
 
     public function testConstructorRejectsNonTypedWriters(): void
     {
-        $this->expectException(
-            version_compare(PHP_VERSION, '7.0.0', '<') ? 'PHPUnit_Framework_Error' : 'TypeError'
-        );
-        $this->expectExceptionMessage(
-            'must implement interface'
```

Comment:
The message changes from php 7 to 8. Since this message does not include any sensitive information, IMO it is better to just remove the check for the message and leave only the type of the exception.